### PR TITLE
Integration test documentation

### PR DESCRIPTION
**Pull request by sanz1995**
I added a few comments in the class IntegrationTest explaining how it works

### CODE REVIEW

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 1]
```diff
@@ -20,10 +20,18 @@
 @DirtiesContext
 public class IntegrationTest {
 
+
+	/**
+	* This class is used to check if the messages that are send to the server contain
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: sent to the server container

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 2]
```diff
@@ -20,10 +20,18 @@
...
+	* what they supposed to have. Integration test should be used after Unit test because
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: they are supposed to

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 3]
```diff
@@ -42,6 +50,14 @@ public void testHome() throws Exception {
 				.getBody().contains("<title>Hello"));
 	}
 	
+	
+	/**  Send a GET message to the server located in localhost and port 0 with the path
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: at localhost*

port 0???

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 4]
```diff
@@ -20,10 +20,18 @@
...
+	* what they supposed to have. Integration test should be used after Unit test because
+	* it tests software modules (witch are alredy correct) as a group.
+	*/
+
+
 	@Value("${local.server.port}")
 	private int port = 0;
 	
-	/**
+	/*
```
Comment (SAReyes) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: Documentation comments should start with double asterisk.

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 5]
```diff
@@ -42,6 +50,14 @@ public void testHome() throws Exception {
 				.getBody().contains("<title>Hello"));
 	}
 	
+	
+	/**  Send a GET message to the server located in localhost and port 0 with the path 
+	* "/webjars/bootstrap/3.3.5/css/bootstrap.min.css" and check if 
+	* the server response is OK (200). It also checks if the response contains 
+	* the text "body", if that´s the case, shows a warming message. If the specified media
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: use `'` apostrophe instead, located under `?` key in spanish keyboard

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java [hunk 6]
```diff
@@ -42,6 +50,14 @@ public void testHome() throws Exception {
 				.getBody().contains("<title>Hello"));
 	}
 	
+	
+	/**  Send a GET message to the server located in localhost and port 0 with the path 
+	* "/webjars/bootstrap/3.3.5/css/bootstrap.min.css" and check if 
+	* the server response is OK (200). It also checks if the response contains 
+	* the text "body", if that´s the case, shows a warming message. If the specified media 
+	* type in the headers is not "text/css", show a warming message.
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: warning*

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java (hunk 3 above)
Comment (SAReyes) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: For consistence purposes, the text should start in a new line under the two asterisks.

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java (hunk 4 above)
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: +1

File: src/test/java/es/unizar/webeng/hello/IntegrationTest.java (hunk 6 above)
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/IntegrationTest.java]: Agree, to be warm with computer is strange.

**Comment by fjlopez** [comment]
You should fix these comments. Sync your repo and write down a note here when your changes are ready to be merged.

**Comment by sanz1995** [comment]
I've just solved what you said and sync my repo, do I have to make a new
pull request?

2016-09-25 17:06 GMT+02:00 Francisco J Lopez-Pellicer <
notifications@github.com>:

**Comment by RecuencoJones** [comment]
@sanz1995 please fix the review comments adding a new commit and then we will approve the Pull Request for merge, no need to create a new pull request

**Comment by sanz1995** [comment]
done

2016-09-25 18:47 GMT+02:00 RecuencoJones <notifications@github.com>:

**Comment by SAReyes** [comment]
Hi @sanz1995, you still need to amend our change request. 

Please push it to your master repo in Github once you are done with it, this PR will be updated automatically.

**Comment by fjlopez** [comment]
LGTM

**Pull request merged**